powerquery-docs/connectors/google-looker.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Looker connector
```

Comment:
```suggestion
title: Looker–Power BI Connector
```

Reply:
Why would this be called "Looker-Power BI connector"? The connector is also supported in Power Apps and Customer Insights (according to Products in the Summary table), which don't have anything to do with Power BI.

Reply:
Apologies, I was coming at this from the Looker-Power BI documentation and I didn't understand that this applies to many products. Please disregard my comment.

Reply:
will keep current name as there may be future plans to support excel